pkgs/by-name/fo/forgejo/generic.nix

```diff
@@ -52,7 +53,7 @@ let
     '';
   };
 in
-buildGoModule rec {
+buildGoModule (finalAttrs: {
```

Comment:
Migrating to `finalAttrs` is unrelated to the fix and also just incomplete.

Please move this out into a separate commit or drop it entirely.

Reply:
I migrated to finalAttrs to simplify the addition of patches from the version specific files. What leads you to believe it's incomplete?